src/containers/ViewerPanel/selectors.ts

```diff
@@ -60,16 +60,18 @@ export const getMaxNumChars = (
     lastFrameTime: number,
     timeStep: number
 ) => {
-    // If lastFrameTime is 15 and step size is 0.2 then 15.2 is probably going to have
-    // the max number of characters for this trajectory
-    const refTimeValue = lastFrameTime + timeStep;
-    const roundedRefTime = roundTimeForDisplay(refTimeValue).toString();
+    // These two time values are likely to have the most digits
+    const refTime1Value = firstFrameTime + timeStep;
+    const refTime2Value = lastFrameTime + timeStep;
+    const roundedRefTime1 = roundTimeForDisplay(refTime1Value).toString();
+    const roundedRefTime2 = roundTimeForDisplay(refTime2Value).toString();
```

Comment:
Is roundTimeForDisplay clamping between firstFrameTime and lastFrameTime?  Just wondering if you might expect lastframe-1 time + timestep to be a good "large" value?

Reply:
What I'm getting at is that I would prefer numbers closer to the last frame rather than the first, as they will hopefully be bigger.

Reply:
Talked with @toloudis . In the long run it would be better to have `roundTimeForDisplay` round to a number of decimal points (which is determined by `stepSize` probably -- and if we don't have any decimal points, `maxNumChars` would simply be `lastFrameTime.length`, I think) rather than to a number of sig figs as it is doing currently. That would probably allow us to determine the `maxNumChars` with more certainty than just guessing. It would also make the time display less chaotic during playback because the length of the time value won't shrink and grow repeatedly. We should discuss this at the next UX meeting.

For now I can't think of another test case that would make `getMaxNumChars` fail, so I think it won't hurt to merge these changes in until we decide exactly what to do in the long run?

Reply:
If we didn't come up with a case to break things, that's probably a good sign.  There's still the question of watching numbers tick by and the decimal point is potentially shifting.

Reply:
It might work well to...
1. Observe how many decimal digits are in the time-step size
2. Fix the display to always display that number of decimal places (fill in '0's when needed)
3. Use (Non-decimal digits in last time-step) + (decimal digits in time-step size) to determine the set size for the time display

This would fix the decimal point in one place, making it easier to tell that the number is continuously increasing (it will visually get longer), and remove some of the guesswork in the display size

Reply:
Agreed @EricJIsaac !